lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestCloseTask.cs

```diff
@@ -55,6 +55,7 @@ public sealed class TestCloseTask : ReefFunctionalTest
         private const string CompletedValidationMessage = "CompletedValidationmessage";
         private const string FailToCloseTaskMessage = "FailToCloseTaskMessage";
         private const string BreakTaskMessage = "BreakTaskMessage";
+        private const string CancelTaskMessage = "CancelTaskMessage";
```

Comment:
I am extremely confused with changes in this test....I do not think these changes were there in earlier commits. Why are these changes part of this PR?